packages/flutter_tools/lib/src/commands/test.dart

```diff
@@ -159,6 +160,17 @@ class TestCommand extends FlutterCommand {
 
     Cache.releaseLockEarly();
 
+    // Run builders once before all tests.
+    if (experimentalBuildEnabled && await flutterProject.hasBuilders) {
+      final CodegenDaemon codegenDaemon = await codeGenerator.daemon(flutterProject);
+      codegenDaemon.startBuild();
+      await for (CodegenStatus status in codegenDaemon.buildResults) {
+        if (status == CodegenStatus.Succeeded || status == CodegenStatus.Failed) {
```

Comment:
Do we not want to issue a warning (like above) if the status is "Failed"?

Reply:
Done